package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "webcomponents.js",
   "version": "1.0.0-rc.8",
-  "description": "webcomponents.js",
+  "description": "Web components polyfill",
```

Comment:
Do polyfills `supports` specifications? Polyfills _implement_ features (specification support) that not supported by browsers natively.

WebComponents.js is more than just a suite, it's a dynamic loader uses feature detection, ES5 adapter, etc...

Anyway, it's much better than original description — `webcomponents.js`.

Feel free to change the description after merge this pull request. But in this case don't forget to change it in `bower.json` file too.